project4/name_server/tasks/bootstrap_task.h

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file Bootstrap Task
+ */
+#ifndef PROJECT4_BOOTSTRAP_TASK_H
+#define PROJECT4_BOOTSTRAP_TASK_H
+
+#include <iostream>
+#include "thread_pool/task.h"
+#include "../bootstrap.h"
+#include "../commands.h"
+#include "../../../common/queue/queue.h"
+
+namespace nameserver_tasks {
```

Comment:
I would recommend you either make this a sub-namespace (e.g. `nameserver::tasks`), or put it all in the `nameserver` namespace.